config/tiktok_cookie.yml.mako

```diff
@@ -0,0 +1,20 @@
+<%!
+    import os
+    def s_v_web_id():
+        return os.getenv('TIKTOK_COOKIE_S_V_WEB_ID', 'verify_kjf974fd_y7bupmR0_3uRm_43kF_Awde_8K95qt0GcpBk')
```

Comment:
The default used to be `'verify_kjf974fd_y7bupmR0_3uRm_43kF_Awde_8K95qt0GcpBk'` but it is not necessary anymore. `None` is better now.